scripts/eosio_build_amazon.sh

```diff
@@ -248,6 +248,105 @@ if [ $? -ne 0 ]; then exit -1; fi
+      && make -j"${JOBS}" && make install \
+      && cd ../ && rm -rf ${TMP_LOCATION}/gcc-7.1.0 ${TMP_LOCATION}/gcc-7.1.0.tar.gz \
+      || exit 1
+   fi
+
+   printf "Checking Clang 8 support...\\n"
+   if [ ! -d $CLANG8_ROOT ]; then
+      printf "Installing Clang 8...\\n"
+      cd ${TMP_LOCATION} \
+      && git clone --depth 1 --single-branch --branch $PINNED_COMPILER_BRANCH https://git.llvm.org/git/llvm.git clang8 && cd clang8 \
+      && git checkout $PINNED_COMPILER_LLVM_COMMIT \
+      && cd tools \
+      && git clone --depth 1 --single-branch --branch $PINNED_COMPILER_BRANCH https://git.llvm.org/git/lld.git \
+      && cd lld && git checkout $PINNED_COMPILER_LLD_COMMIT && cd ../ \
+      && git clone --depth 1 --single-branch --branch $PINNED_COMPILER_BRANCH https://git.llvm.org/git/polly.git \
+      && cd polly && git checkout $PINNED_COMPILER_POLLY_COMMIT && cd ../ \
+      && git clone --depth 1 --single-branch --branch $PINNED_COMPILER_BRANCH https://git.llvm.org/git/clang.git clang && cd clang/tools \
+      && git checkout $PINNED_COMPILER_CLANG_VERSION \
+      && mkdir extra && cd extra \
+      && git clone --depth 1 --single-branch --branch $PINNED_COMPILER_BRANCH https://git.llvm.org/git/clang-tools-extra.git \
+      && cd clang-tools-extra && git checkout $PINNED_COMPILER_CLANG_TOOLS_EXTRA_COMMIT && cd .. \
+      && cd ../../../../projects \
+      && git clone --depth 1 --single-branch --branch $PINNED_COMPILER_BRANCH https://git.llvm.org/git/libcxx.git \
+      && cd libcxx && git checkout $PINNED_COMPILER_LIBCXX_COMMIT && cd ../ \
+      && git clone --depth 1 --single-branch --branch $PINNED_COMPILER_BRANCH https://git.llvm.org/git/libcxxabi.git \
+      && cd libcxxabi && git checkout $PINNED_COMPILER_LIBCXXABI_COMMIT && cd ../ \
+      && git clone --depth 1 --single-branch --branch $PINNED_COMPILER_BRANCH https://git.llvm.org/git/libunwind.git \
+      && cd libunwind && git checkout $PINNED_COMPILER_LIBUNWIND_COMMIT && cd ../ \
+      && git clone --depth 1 --single-branch --branch $PINNED_COMPILER_BRANCH https://git.llvm.org/git/compiler-rt.git \
+      && cd compiler-rt && git checkout $PINNED_COMPILER_COMPILER_RT_COMMIT && cd ../ \
+      && cd ${TMP_LOCATION}/clang8 \
+      && mkdir build && cd build \
+      && $CMAKE -G "Unix Makefiles" -DCMAKE_INSTALL_PREFIX="${CLANG8_ROOT}" -DLLVM_BUILD_EXTERNAL_COMPILER_RT=ON -DLLVM_BUILD_LLVM_DYLIB=ON -DLLVM_ENABLE_LIBCXX=ON -DLLVM_ENABLE_RTTI=ON -DLLVM_INCLUDE_DOCS=OFF -DLLVM_OPTIMIZED_TABLEGEN=ON -DLLVM_TARGETS_TO_BUILD=all -DCMAKE_BUILD_TYPE=Release .. \
```

Comment:
> -DLLVM_TARGETS_TO_BUILD=all

Is it worth making this `-DLLVM_TARGETS_TO_BUILD=host` to speed up compile for users or not?